backend/api/Database/Models/Robot.cs

```diff
@@ -28,6 +28,8 @@ public class Robot
 
         public string CurrentAsset { get; set; }
 
+        public AssetDeck? CurrentAssetDeck { get; set; }
```

Comment:
Is this nullable?

Reply:
Yes for now. It will be reintroduced in a different way in a different PR. It can be made non-nullable not sure if it adds any value ?

Reply:
It should be not required incase we make an instance of robot when it is not localised yet, because then we have to make a dummy key and key is supposed to be unique. Key is a variable inside AssetDeck class.